utils/_context/containers.py

```diff
@@ -162,10 +164,13 @@ def start(self) -> Container:
         self.wait_for_health()
         self.warmup()
 
+        if not replay:
```

Comment:
I don't get your code. You always starts the container (in both normal mode and replay mode), but you execute `_post_start` only on replay mode?  🤔

Reply:
Looks like I did that in reverse, pushing a fix now.

Reply:
Ended up just reverting the change to `post_start` as the main goal of this PR was addressing `start`.